_tutorials/2016-05-10-iosxr-ansible.md

```diff
@@ -148,33 +152,80 @@ commit
 end
 ```
 
-Finally, let's check connectivity between boxes:   
-
+Checking connectivity between boxes:
 ```
-RP/0/RP0/CPU0:ios#ping 10.1.1.20
+RP/0/RP0/CPU0:ios#ping 10.1.1.10
 Mon May  9 08:36:33.071 UTC
 Type escape sequence to abort.
-Sending 5, 100-byte ICMP Echos to 10.1.1.20, timeout is 2 seconds:
+Sending 5, 100-byte ICMP Echos to 10.1.1.10, timeout is 2 seconds:
 !!!!!
 Success rate is 100 percent (5/5), round-trip min/avg/max = 1/5/20 ms
 RP/0/RP0/CPU0:ios#
 ```
 
-Let's copy public part of key from Ubuntu box and allow access without password. For IOS-XR image 6.1.x commands below:   
 
+Let's copy public part of key from **Ubuntu** box and allow access without password. At first we should connect to Ubuntu and copy file to XR via SCP
 ```
-vagrant port
+vagrant ssh ubuntu
+scp -P 57722 /home/vagrant/.ssh/id_rsa.pub  vagrant@<XR's interface address>:/home/vagrant/id_rsa_ubuntu.pub
 
```

Comment:
When you execute this command, perform 2 steps:

First, let's transfer the ubuntu public key to XR linux shell, to enable passwordless access to XR linux.
```
scp -P 57722 /home/vagrant/.ssh/id_rsa.pub  vagrant@<XR's interface address>:/home/vagrant/id_rsa_ubuntu.pub
```

Now, let's transfer the base64 version of the ubuntu public key to enable passwordless access to XR CLI over ssh.
```
scp -P 57722 /home/vagrant/.ssh/id_rsa_pub.b64  vagrant@<XR's interface address>:/disk0:/id_rsa_pub.b64
```